addons/mrp_account/wizard/post_wip.py

```diff
@@ -0,0 +1,120 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from datetime import timedelta
+
+from odoo import fields, models, _, api
+from odoo.exceptions import UserError
+
+
+class PostWipLine(models.TransientModel):
+    _name = 'mrp.account.post.wip.line'
+    _description = 'Line to be created for posting WIP'
+
+    account_id = fields.Many2one('account.account')
+    label = fields.Char()
+    debit = fields.Monetary()
+    credit = fields.Monetary()
+    currency_id = fields.Many2one('res.currency', default=lambda self: self.env.company.currency_id)
+    post_wip_id = fields.Many2one('mrp.account.post.wip')
+
+    _sql_constraints = [
+        ('check_debit_credit', 'CHECK ( debit = 0 OR credit = 0 )',
+         'A single line cannot be both credit and debit.')
+    ]
+
+    @api.onchange('debit')
+    def _inverse_debit(self):
+        for line in self:
+            if line.debit:
+                line.credit = 0
+
+    @api.onchange('credit')
+    def _inverse_credit(self):
+        for line in self:
+            if line.credit:
+                line.debit = 0
+
+
+class PostWip(models.TransientModel):
+    _name = 'mrp.account.post.wip'
+    _description = 'Post Manufacturing WIP'
+
+    date = fields.Date()
+    reversal_date = fields.Date()
+    journal_id = fields.Many2one('account.journal')
+    reference = fields.Char()
+    line_ids = fields.One2many('mrp.account.post.wip.line', 'post_wip_id')
+
+    @api.model
+    def default_get(self, fields_list):
```

Comment:
default_get is before the declaration of fields https://www.odoo.com/documentation/17.0/contributing/development/coding_guidelines.html#symbols-and-conventions